tools/release_checks.rb

```diff
@@ -46,8 +42,8 @@ def check_tag
     end
   end
 
-  def check_jenkins_state
-    uri = URI.parse("https://ci.opensuse.org/job/#{@jenkins_name}/lastStableBuild/api/json")
+  def check_jenkins_state(jenkins_url)
```

Comment:
Assignment Branch Condition size for check_jenkins_state is too high. [30.51/15]